src/core_plugins/metrics/public/visualizations/lib/events.js

```diff
@@ -0,0 +1,3 @@
+import $ from 'jquery';
+export default $({});
```

Comment:
This is using jQuery to create an  event bus, it's being used to synchronize the crosshairs on the time series chart